src/build.sh

```diff
@@ -150,6 +159,21 @@ function parse_build_options()
         options_values['MENU_CONFIG']="${build_config[menu_config]:-nconfig}"
         shift
         ;;
+      --cpu-scaling | -c)
+        if [[ ! "$2" =~ [0-9]+ ]]; then
+          options_values['ERROR']="$2"
+          return 22 # EINVAL
+        fi
+        if [[ "$2" -gt 100 ]]; then
+          complain 'Upscaling CPU performance in compilation tasks may have unintended consequences!'
+        fi
```

Comment:
@kwy95 I think we discussed something like this in the `kw mail`. Could you review this part?

Reply:
@kwy95 ping

Reply:
Hi, the relevant function in the mail.sh file is `reposition_commit_count_arg` and the end of the parser as well, if you want take a look at it. I don't think the use case is the same, but it might help somehow. Anyway, I will review this section further in the new PR.